azure_functions_worker/bindings/meta.py

```diff
@@ -89,11 +118,21 @@ def is_trigger_binding(bind_name: str) -> bool:
 def check_input_type_annotation(bind_name: str,
                                 pytype: type,
                                 is_deferred_binding: bool) -> bool:
+    global INPUT_TYPE_CHECK_OVERRIDE_MAP
+    if bind_name in INPUT_TYPE_CHECK_OVERRIDE_MAP:
+        return INPUT_TYPE_CHECK_OVERRIDE_MAP[bind_name](bind_name, pytype,
+                                                        is_deferred_binding)
```

Comment:
HTTP isn't a supported deferred binding type. You don't need to send in is_deferred_binding, it will default to false in get_binding

Reply:
i added it for future extensibility in case override map support blobs or any types later. its fine to pass as http binding case at runtime is_deferred_binding is always false